string_permutation_problem/permute.py

```diff
@@ -0,0 +1,23 @@
+def permute_my_string(str):
+    output_string = []
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpu5cefre7/string_permutation_problem/permute.py
+++ b/tmp/tmpu5cefre7/string_permutation_problem/permute.py
@@ -4,7 +4,7 @@
         This is a recursive solution to the string permutation problem.
         The user is supposed to return all the permutations for a given string.
     """
-    #Base Case
+    # Base Case
     if len(str) == 1:
         output_string = [str]
     else:
```